cg/meta/archive/ddn_dataflow.py

```diff
@@ -39,9 +39,13 @@ class MiriaFile(FileTransferData):
     source: str
 
     @classmethod
-    def from_file_and_sample(cls, file: File, sample: Sample) -> "MiriaFile":
+    def from_file_and_sample(
```

Comment:
This method is confusing to me, how can the sample internal id be the destination? Will a sample model with the provided id be populated with data from the file?

Reply:
@seallard It is basically down to what kind of file structure we want in the data lake we will be storing the data in. Current suggestion is to have a very simple structure wherein we put the files within a directory which we simply call the same as the sample_internal_id. Would love to test exactly how this works once the API is up, though.

In the request, the source is where the file is currently located and destination is where we want to put it. I think for retrieving files we will probably need to think about this a bit - are we fetching a folder and putting it in the location where there previously was a file? Maybe we should drop everything after "version" in the destination string at that point. Probably no point in developing this until their API is up.